pkgs/servers/haste-server/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, nixosTests
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, nodejs
+, pkgs
+}:
+
+stdenv.mkDerivation rec {
+  pname = "haste-server";
+  version = "3dcc43578b99dbafac35dece9d774ff2af39e8d0";
```

Comment:
This should be in the unstable-XXXX-XX-XX format. Also please update the PR title.

Reply:
See https://github.com/NixOS/nixpkgs/pull/162509#issuecomment-1057166504

If you figure out how to make update script be able to convert date into ref and back, go for it, but I for one think that's lunacy.

Reply:
That is pretty easy.

```ShellSession
$ curl -s "https://api.github.com/repos/toptal/haste-server/commits?per_page=1" | jq -r ".[0] | .sha, .commit.author.date"
3dcc43578b99dbafac35dece9d774ff2af39e8d0
2022-02-14T19:48:23Z
```

Reply:
I mean how do I check if the version is up to date then? given yyyy-mm-dd there might be multiple commits mapping that version.